chrome_extension/vendor/mooltipass/mcCombinations.js

```diff
@@ -140,6 +140,15 @@ var extendedPost = {
 	}
 }
 
+var extendedSubmitButton = {
+	'vodafone.de': function(form) {
+		return form.parent().parent().find('#loginBtn')
```

Comment:
Maybe later we can rely on buttons and not on form submit event and it will cover all cases when scripts handle submit event only. Need to run tests and look how it will be.